tasks/__init__.py

```diff
@@ -426,28 +426,42 @@ def flake():
 
 
 @task(aliases=['req'])
-def requirements(addons=False, release=False, dev=False, metrics=False):
+def requirements(base=False, addons=False, release=False, dev=False, metrics=False, quick=False):
     """Install python dependencies.
 
     Examples:
+        inv requirements
+        inv requirements --quick
 
-        inv requirements --dev
-        inv requirements --addons
-        inv requirements --release
-        inv requirements --metrics
+    Quick requirements are, in order, addons, dev and the base requirements. They should do for day-to-day development
+    work so you get everything that someone might have added that would keep you from running the software properly.
+
+    By default, base requirements will run. However, if any set of addons, release, dev, or metrics are chosen, base
```

Comment:
And yes, if you set `--dev --metrics` then you would only run the dev and metrics branches.

Most of the complication comes from the backwards compatibility for the release branch, incidentally. It would have been much simpler otherwise.